src/main/java/com/atlassian/bitbucket/jenkins/internal/provider/JenkinsProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.atlassian.bitbucket.jenkins.internal.provider;
+
+import jenkins.model.Jenkins;
+
+import javax.annotation.Nonnull;
+
```

Comment:
if we want Guice to be able to inject this we also need a `@ImplementedBy(` annotation on it.